packages/runner/src/types/tasks.ts

```diff
@@ -498,14 +498,14 @@ export type HookListener<T extends any[], Return = void> = (
   ...args: T
 ) => Awaitable<Return>
 
-export type HookCleanupCallback = (() => Awaitable<unknown>) | void
+export type HookCleanupCallback = (() => Awaitable<unknown>) | any
```

Comment:
I would prefer if we used `unknown` instead.

`(() => Awaitable<unknown>) | any` or `(() => Awaitable<unknown>) | unknown` also don't make sense because both `unknown` and `any` have higher priority, so a single `unknown` should be fine

Reply:
Hi @sheremet-va 👋
Do we still need to keep the 'export type HookCleanupCallback = unknown' type, or can we just replace all `HookCleanupCallback` with 'unknown'?

Reply:
We should just deprecate it because removing the export is a breaking change

Reply:
Ready for review! 🚀